src/ResourceManager/Profile/Commands.Profile/Context/ImportAzureRMContext.cs

```diff
@@ -0,0 +1,100 @@
+using Microsoft.Azure.Commands.Profile.Properties;
+using Microsoft.Azure.Commands.ResourceManager.Common;
+using Microsoft.WindowsAzure.Commands.Common;
+using System;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Profile
+{
+    /// <summary>
+    /// Selects Microsoft Azure profile.
+    /// </summary>
+    [Cmdlet(VerbsData.Import, "AzureRmContext", SupportsShouldProcess = true), OutputType(typeof(PSAzureProfile))]
+    [Alias("Select-AzureRmProfile")]
+    [Obsolete("Select-AzureRmProfile will be renamed to Import-AzureRmContext in the next release.")]
+    public class ImportAzureRMContextCommand : AzureRMCmdlet
+    {
+        internal const string InMemoryProfileParameterSet = "InMemoryProfile";
+        internal const string ProfileFromDiskParameterSet = "ProfileFromDisk";
+
+        [Parameter(ParameterSetName = InMemoryProfileParameterSet, Mandatory = true, Position = 0, ValueFromPipelineByPropertyName = true)]
+        [ValidateNotNull]
+        [Alias("Profile")]
+        public AzureRMProfile AzureContext { get; set; }
+
+        [Parameter(ParameterSetName = ProfileFromDiskParameterSet, Mandatory = true, Position = 0, ValueFromPipelineByPropertyName = true)]
+        [ValidateNotNullOrEmpty]
+        public string Path { get; set; }
+
+        protected override void BeginProcessing()
+        {
+            // Do not access the DefaultContext when loading a profile
+        }
+
+        public override void ExecuteCmdlet()
+        {
+            bool executionComplete = false;
+            if (MyInvocation.BoundParameters.ContainsKey("Path"))
+            {
+                ConfirmAction(string.Format(Resources.ProcessImportContextFromFile, Path), Resources.ImportContextTarget, () =>
+                {
+                    if (!Common.Authentication.AzureSession.DataStore.FileExists(Path))
+                    {
+                        throw new PSArgumentException(string.Format(
+                            Microsoft.Azure.Commands.Profile.Properties.Resources.FileNotFound,
+                            Path));
+                    }
+
+                    AzureRmProfileProvider.Instance.Profile = new AzureRMProfile(Path);
+                    executionComplete = true;
+                });
+            }
+            else
+            {
+                ConfirmAction(Resources.ProcessImportContextFromObject, Resources.ImportContextTarget, () =>
+                {
+                    AzureRmProfileProvider.Instance.Profile = AzureContext;
+                    executionComplete = true;
+                });
+            }
+
+            if (executionComplete)
+            {
+                if (AzureRmProfileProvider.Instance.Profile == null)
+                {
+                    WriteExceptionError( new ArgumentException(Resources.AzureProfileMustNotBeNull));
+                }
+
+                if (AzureRmProfileProvider.Instance.Profile.Context != null &&
```

Comment:
We don't have access to this operator, because we are not using that version of the compiler.  And again, this is just moved existing code.